terraform/groups/ecs-service/locals.tf

```diff
@@ -7,7 +7,7 @@ locals {
   container_port             = "8080" # default Java port to match start script
   docker_repo                = "acsp-manage-users-api"
   lb_listener_rule_priority  = 20
-  lb_listener_paths          = ["/acsp-manage-users-api/healthcheck, /acsp-members/*, /internal/acsp-members/active"]
+  lb_listener_paths          = ["/acsp-manage-users-api/healthcheck", "/acsp-members/*", "/internal/acsp-members/active"]
```

Comment:
hi this needs changing, /internal/acsp-members/active to /internal/acsp-members/*